spring-boot-project/spring-boot-docs/src/docs/asciidoc/actuator/metrics.adoc

```diff
@@ -175,9 +175,13 @@ Please ensure that the endpoint URI contains the path (e.g. `/api/v2/metrics/ing
 	  metrics:
 	    export:
 	      dynatrace:
-	        # uri: "https://{your-domain}/e/{your-environment-id}/api/v2/metrics/ingest" for managed deployments.
-	        uri: "https://{your-environment-id}.live.dynatrace.com/api/v2/metrics/ingest"
-	        api-token: "YOUR_TOKEN" # should be read from a secure source and not hard-coded.
+	        # uri: "https://{your-environment-id}.live.dynatrace.com/api/v2/metrics/ingest" # for SaaS
+	        # uri: "https://{your-domain}/e/{your-environment-id}/api/v2/metrics/ingest"    # for managed deployments
```

Comment:
I am not keen to provide multiple options in the same example. These are not rendered for properties anyway and it feels like this is something that should be described rather than being put in the example directly.

Reply:
The idea here was that users could just copy the example and remove the # for the option they want to use.

Reply:
OK but that doesn't address my previous comment.

Reply:
Oh, I might have missed something here. Do I understand correctly that the comments within the code blocks will not show up in the documentation at all?

Reply:
@snicoll, Is there something I missed here? Would you be able to provide some guidance on how to proceed?

Reply:
>  Do I understand correctly that the comments within the code blocks will not show up in the documentation at all?

There are not for properties. You can figure that out yourself by looking at the [current documentation](https://docs.spring.io/spring-boot/docs/2.6.0-M2/reference/html/actuator.html#actuator.metrics.export.dynatrace.v2-api). We render the snippet in both `yml` and `properties` format. The latter doesn't have the comment. Ideally I'd like to rework those samples irrespective of this PR to remove the comment so I am not keen to add more comments.

> Would you be able to provide some guidance on how to proceed?

From my perspective, it boils down to my first comment, specifically "it feels like this is something that should be described rather than being put in the example directly.". If we feel these are important enough examples to document, the different options should be described in text with some context, rather than in a comment of a snippet. I've flagged for team attention to get more feedback from the team.

Reply:
In 57bd19f I moved the additional information to text paragraphs, either way is fine with me. The comments that still persist are optional, meaning that users that copy from the `properties` format still get the expected behavior, but in the `yml` format, they provide additional information.

Reply:
We should aim to treat users equally, irrespective of the format that they choose to use for configuration files. If the additional information is needed, I don't think it should be hidden in a comment that's only visible if you're using YAML.

Reply:
@wilkinsona I have removed all comments, except for one that is only relevant for users of the YAML format. Thanks for explaining this, I did not realize that they would not show up in the properties file.